### PR TITLE
Add descriptions to other parameters.

### PR DESCRIPTION
**Pull request by oasaleh**
This change adds descriptions to **Delay** and **For** parameters.

### CODE REVIEW

File: docset/winserver2022-ps/hyper-v/Restart-VM.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ Accept wildcard characters: False
 ```
 
 ### -Delay
-
+Specifies the frequency of queries, in seconds. PowerShell queries the service specified by the For parameter to determine whether the service is available after the computer is restarted. This parameter is valid only together with the `-Wait` and `-For` parameters. If the Delay parameter isn't specified, **Restart-VM** uses a five second delay.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/Restart-VM.md:144]:
```suggestion
Specifies the frequency of queries, in seconds. PowerShell queries the service specified by the `-For` parameter to determine whether the service is available after the computer is restarted. This parameter is valid only with the `-Wait` and `-For` parameters. If the `-Delay` parameter isn't specified, **Restart-VM** uses a five-second delay.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @oasaleh, kindly commit the minor suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
Can this be added to other OS versions of the article? Thanks

**Comment by scanum** [comment]
Hi @tiburd hope all is well. Could you please help with this PR? Thanks!

**Comment by tiburd** [comment]
@oasaleh 
Hi Omar,

To help ensure content health and consistency between operating system versions, please add your proposed changes for all operating system versions.

It looks like these are the other affected operating system versions:
- winserver2012-ps
- winserver2016-ps
- winserver2019-ps
- winserver2012r2-ps

After you finish that, please @ mention me or @scanum so we can have the changes reviewed and approved.

Thank you!

cc: @scanum

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**